assets/js/tour.js

```diff
@@ -129,13 +129,18 @@ document.addEventListener( 'DOMContentLoaded', function () {
 				}
 				wrapper.classList.add( 'pulse-wrapper' );
 			}
+
 			if ( ! wrapper.querySelector( '.pulse' ) ) {
-				const pulse = document.createElement( 'div' );
+				const pulse = document.createElement( 'button' );
+				pulse.dataset.tourTitle =
+					tour_plugin.tours[ tourId ][ 0 ].title;
+				pulse.ariaLabel =
```

Comment:
The aria label should match any visible version of the same text. If other visible text is just the name of the tour, that should also be true of the aria label. There's no need to give screen reader users information that other users don't have access to.

Reply:
There is no user visible text here, just the glowing dot. If we'd give it a visible title, the `ariaLabel` is what I'd give it. So I'd suggest to use this ariaLabel.